src/miner_jsonrpc_handler.erl

```diff
@@ -58,6 +58,8 @@ handle_rpc_(<<"sc_", _/binary>> = Method, Params) ->
     miner_jsonrpc_sc:handle_rpc(Method, Params);
 handle_rpc_(<<"peer_", _/binary>> = Method, Params) ->
     miner_jsonrpc_peer:handle_rpc(Method, Params);
+handle_rpc_(<<"version", _/binary>> = Method, Params) ->
```

Comment:
Would this not be better hanging of of info_ ?

Reply:
Good point.